tensorflow/lite/micro/tools/make/helper_functions.inc

```diff
@@ -505,6 +506,8 @@ $$($(1)_BINARY): $$($(1)_LOCAL_OBJS) $$(MICROLITE_LIB_PATH)
 $(1): $$($(1)_BINARY)
 $(1)_bin: $$($(1)_BINARY).bin
 
+GENERATOR_INPUTS += $(4)
```

Comment:
might be worth a comment that we only add to the list here and the actual generation happens later to prevent repeating the same generation multiple times for different build targets.

Reply:
Ack - added generation here.